hera_pspec/pspecdata.py

```diff
@@ -1545,21 +1583,9 @@ def get_Q(self, mode, pol=False):
         tau = self.delays()[int(mode)] * 1.0e-9 # delay in seconds
         nu  = self.freqs[self.spw_range[0]:self.spw_range[1]] # in Hz
 
-        try:
-            beam_res, beam_omega, N = self.primary_beam.beam_normalized_response(pol, nu) 
-            #Get beam response in (frequency, pixel), beam area(freq) and Nside, used in computing dtheta.
-            prod          = (1.0/beam_omega) 
-            beam_prod     = beam_res * prod[:, np.newaxis] 
-            integral_beam = (np.pi/(3.0*(N)**2))* \
-                                      np.dot(beam_prod, beam_prod.T) #beam_prod has omega subsumed, but taper is still part of R matrix
-                                                                     # the nside terms is dtheta^2, where dtheta is the resolution in healpix map
-        except(AttributeError):
-            warnings.warn('The beam response could not be calculated. PS will not be normalized!')
-            integral_beam = np.ones((len(nu), len(nu)))
-
         eta_int = np.exp(-2j * np.pi * tau * nu) #exponential part of the expression
         Q_alt   = np.einsum('i,j', eta_int.conj(), eta_int) # dot it with its conjugate
-        Q       = Q_alt * integral_beam 
```

Comment:
Related to this, it would also be useful to mention the improvement in speed after this refactoring (may be at the documentation of pspec fucntion (exact_norm parameter).